ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/LgrDB.hs

```diff
@@ -236,7 +236,7 @@ initFromDisk
      , HasCallStack
      )
   => LgrDbArgs Identity m blk
-  -> Tracer m (TraceReplayEvent blk ())
+  -> Tracer m (TraceReplayEvent blk (Point blk, ()))
```

Comment:
Please change this argument type to `TraceReplayEvent blk (ReplayStart blk)`

Or to `ReplayGoal blk -> TracerReplayEvent blk`, if we go with my new suggestion.

Reply:
Done